### ISSUE TITLE
Missing UI for setting the system prompt

### ISSUE THREAD

**Issue by samie** [ollama4j/ollama4j-web-ui#6]
For chat to be useful for testing there should be a configuration for setting the LLM system prompt. 

Couple of ideas: 
- Cogwheel button for configuration in top of the chat view that opens a text area dialog
- Menu item for LLM configuration that can support list of predefined system prompts

There also should be a storage to persist the system prompts and settings.

**Comment by amithkoujalgi** [comment]
Thanks @samie, sounds like a great idea. But I am not so well versed with Vaadin setup. I'll have to look around to see how to accomplish this.

**Comment by samie** [comment]
I can definitely help with the UI part. Just good to align and agree about the goals before writing any code.

**Comment by amithkoujalgi** [comment]
Absolutely! I appreciate your willingness to help. It would definitely be great to make these updates. Looking forward to collaborating!
